src/bin/bat/app.rs

```diff
@@ -287,6 +287,7 @@ impl App {
             use_custom_assets: !self.matches.get_flag("no-custom-assets"),
             #[cfg(feature = "lessopen")]
             use_lessopen: self.matches.get_flag("lessopen"),
+            set_terminal_title: self.matches.get_flag("set_terminal_title"),
```

Comment:
Using hypens instead of underscores (i.e. `--set-terminal-title`) would make the option more consistent with the rest of `bat`'s options.